recipes/lsst-documenteer-technote/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "documenteer" %}  {# PyPI name #}
+{% set version = "0.5.6" %}
+
+package:
+  name: "lsst-documenteer-technote"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "06fbceb7324ed3b00f012b90b2da0e0aff9faab7957d6d7c5f09cc5a3069c9cf"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - refresh-lsst-bib = documenteer.bin.refreshlsstbib:run
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
You shouldn't need the extra install, right?

Reply:
Aha, done.